RELEASE.md

```diff
@@ -6,23 +6,27 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 1. Merge all PRs intended for the release into the `master` branch
 1. Checkout and update the master branch and ensure all tests are passing:
-	* `git checkout master`
-	* `git pull`
+    * `git checkout master`
+    * `git pull`
+    * install and update necessary dependencies:
+        * go get -v -u honnef.co/go/tools/cmd/staticcheck
+        * go get -v -u golang.org/x/lint/golint
+        * go get -v -u github.com/securego/gosec/cmd/gosec
     * `make all`
 1. Update the [`CHANGELOG.md`](https://github.com/fastly/cli/blob/master/CHANGELOG.md):
-	* Apply necessary labels (`enchancement`, `bug`, `documentation` etc) to all PRs intended for the release that you wish to appear in the `CHANGELOG.md`
-	* **Only add labels for relevant changes**
+    * Apply necessary labels (`enchancement`, `bug`, `documentation` etc) to all PRs intended for the release that you wish to appear in the `CHANGELOG.md`
+    * **Only add labels for relevant changes**
     * `git checkout -b vx.x.x` where `vx.x.x` is your target version tag
     * `make changelog`
     * `git add CHANGELOG.md && git commit -m "vX.X.X"`
 1. Send PR for the `CHANGELOG.md`
 1. Once approved and merged, checkout and update the `master` branch:
-	* `git checkout master`
-	* `git pull`
+    * `git checkout master`
+    * `git pull`
 1. Create a new tag for `master`:
-	* `git tag -s vx.x.x -m "vx.x.x"`
+    * `git tag vx.x.x -m "vx.x.x"`
```

Comment:
Signed commits for the tag releases was something which was advised by our internal security team as a good best practice for releasing. Personally i'd prefer it if we adhered to this. So maybe we should add a comment to the top suggesting to configure git correctly?

Reply:
I removed this commit from the branch 👍